src/js/jsx/sections/style/Type.jsx

```diff
@@ -142,12 +142,16 @@ define(function (require, exports, module) {
         },
 
         /**
-         * Set the type face of the selected text layers from a font's postscript name.
+         * Set the type face of the selected text layers from a font's ID.
          * 
          * @private
-         * @param {string} postScriptName
+         * @param {string} typefaceID
          */
-        _handleTypefaceChange: function (postScriptName) {
+        _handleTypefaceChange: function (typefaceID) {
+            var typefaceIndex = parseInt(typefaceID), // typefaceID is equivalent to index in this.state.typefaces
```

Comment:
Drive-by nit: we should never use `window.parseInt`. It's harmless here, but in most other cases the fact that it returns a `NaN` when the number doesn't parse is a real bummer. (We could probably talk JSHint or JSCS into flagging this as an error.) Try `parseNumber` from `js/util/math` instead.